articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,194 @@
+
+Proceed to sanitize the UDF files the attacker used to obtain the OS Shell. In addition, change your encrypted password to more stronger values. Finally, check if the port 3306 is open on the internet and if so, block it.
+
+##### Fix WordPress SQLi Attack: Database Clean Up
+
+A database clean-up is required to fix any possible SQLi on your site's database. Here are steps to so as well with the function cheat-sheet.
+
+Step 1: Look up all your tables using the command show tables; Look out for a table named Sqlmap.
+
+Step 2: If the table Sqlmap is present, it has been used against your website. Delete the Sqlmap table using the command 
+~~~~sql
+DROP TABLE Sqlmap;
+~~~~
+
+Step 3: Look out for new/foreign users in your db using the following code
+
+~~~~sql
+Select * FROM users  AS u AND u.created > UNIX_TIMESTAMP(STR_TO_DATE('sept 22 2020', '%M %d %Y '));
+~~~~
+Manipulate the string in the code to determine when any of the users found were created.
+
+Step 4: Delete any rogue users using the command below:
+
+~~~~sql
+DROP USER 'malicious'@'localhost';
+~~~~
+
+Step 5: Change and secure your passwords using the command below.
+~~~~sql
+UPDATE users SET pass = concat('ZZZ', sha(concat(pass, md5(rand()))));
+~~~~
+
+##### Fix WordPress SQLi Attack: Vital updates
+
+Updates for your WordPress site are very vital in keeping ahead of day to day vulnerabilities. Make sure to update your WordPress Core files oftenly as well as your theme and plugins. Avoid using nulled themes on your WP site. In addition, make sure to use reputable plugins on your site.
+
+##### Fix WordPress SQLi Attack: Use a Firewall
+
+A firewall detects and shields your website from SQLi as well other attacks depending on the features of your firewall service provider. There are quite of firewall security solutions on the internet; look out for one that suits your WP security needs.
+
+##### Fix WordPress SQLi Attack: Pentesting
+
+[Pentesting/penetration testing](https://www.cloudflare.com/learning/security/glossary/what-is-penetration-testing/) or rather website security audit is a vital operation to keep your WP site away from possible SQLi attacks. WordPress Pentesting involves doing simulated SQLi attacks on your site to make all loopholes are covered. This process ensures all loopholes are found and sealed before an attacker finds and takes advantage of the same.
+
+### How to Prevent SQLi in WordPress
+
+As they say, prevention is better than cure. Don't wait to prove if that is right or wrong with a successful SQLi on your WordPress website. Proper site maintenance practices will save you from SQLi attacks. These practices include regular updates for WordPress core files, plugins updates, use of reputable plugins, avoid nulled themes and minimal user inputs fields as much as possible.
```

Comment:
if that **proverb** would make your point clearer.
**minimal user inputs fields** isn't very clear. Please revise.

Reply:
Can you take a look at this please, @jwk19.